documentation/_include/podman.adoc

```diff
@@ -64,7 +64,7 @@ Linux containers are technologies (plural) that allow you to package and isolate
   * improve flexibilty over virtual machines
 
 Because container images include only the content needed to run an application, a container is more efficient 
-requires less to run.  Likewise, since the container is not running the entirety of a complete operating system,
+requires less to run.  Likewise, since the container is not running complete operating system,
```

Comment:
This does not feel accurate for me.  How about:  Likewise, since a containerized application does not include (or execute) a complete operating system stack,  it will typically run faster than one that carries with it the overhead of a whole new virtual 
machine.

Reply:
It doesn't help I left out a word between running and complete. Apologies.
My intent was "Likewise, since the container is not running **a** complete operating system,"

My read on the use of both entirety and complete in that sentence was that the two were redundant where one of those could suffice.